megatron/arguments.py

```diff
@@ -621,6 +621,8 @@ def _add_learning_rate_args(parser):
     group.add_argument('--lr-decay-tokens', type=int, default=None,
                        help='number of tokens to decay learning rate over,'
                        ' If not None will override iter/sample-based decay')
+    group.add_argument('--lr-warmup-style', type=str, default='linear',
+                       choices=['constant', 'linear']),
```

Comment:
You can probably add some help documentation, in order to know what does `constant` warmup for example?